build.fsx

```diff
@@ -1,3 +1,5 @@
+System.IO.Directory.SetCurrentDirectory __SOURCE_DIRECTORY__
+
```

Comment:
To be honest I don't even remember why its needed (I'm pretty sure the author of this). Now you are :)